resources/js/tall-stack/dynamic/reorderSiteAwards/handleResetOrder.ts

```diff
@@ -0,0 +1,52 @@
+import { collectMappedTableRows } from './collectMappedTableRows';
+import type { MappedTableRow } from './index';
+import { computeDisplayOrderValues } from './index';
+
+declare global {
+  interface Window {
+    postAllAwardsDisplayOrder: (awards: Partial<MappedTableRow>[]) => void;
+  }
+}
+
+export function handleResetOrder(): void {
+  if (!confirm('This will reset the order of all awards. Are you sure?')) {
```

Comment:
Ambiguity: this will indeed reset the order, but could be misinterpreted to mean "it will undo the changes I've made in my current session". I recommend a more precise confirmation:

> This will resort all your awards by the date they were earned (oldest first). You can preview the changes before saving.

The "preview" bit is related to another comment I'll leave in the review.